datajunction-server/datajunction_server/construction/build.py

```diff
@@ -1461,6 +1467,13 @@ def build_materialized_cube_node(
                     for proj in measures_combiner_ast.select.projection
                 ],
             )
+        else:
+            combined_ast.select.projection.append(
+                ast.Function(
+                    name=ast.Name("SUM"),
```

Comment:
This is the materialized metrics table case. It seems sensible to just choose `SUM` for now, since there shouldn't be any other possible aggregation types on a metric (maybe `MAX` or `MIN` in some special cases).

Reply:
I think what you said here should be a code comment.